frontend/src/metabase/admin/settings/components/SettingsPages/UpdatesSettingsPage.unit.spec.tsx

```diff
@@ -0,0 +1,166 @@
+import userEvent from "@testing-library/user-event";
+import fetchMock from "fetch-mock";
+import { act } from "react-dom/test-utils";
+
+import {
+  setupPropertiesEndpoints,
+  setupSettingsEndpoints,
+  setupUpdateSettingEndpoint,
+} from "__support__/server-mocks";
+import { renderWithProviders, screen, waitFor } from "__support__/ui";
+import { UndoListing } from "metabase/containers/UndoListing";
+import type { SettingKey, UpdateChannel } from "metabase-types/api";
+import {
+  createMockSettingDefinition,
+  createMockSettings,
+} from "metabase-types/api/mocks";
+
+import { UpdatesSettingsPage } from "./UpdatesSettingsPage";
+
+const setup = async (props: {
+  updateChannel: UpdateChannel;
+  isHosted: boolean;
+  versionTag: string;
+}) => {
+  const updatesSettings = {
+    "is-hosted?": props.isHosted,
+    "check-for-updates": true,
+    "update-channel": "latest",
+    version: {
+      date: "2025-03-19",
+      src_hash: "4df5cf3e5e86b0cc7421d80e2a8835e2ce3afa7d",
+      tag: props.versionTag,
+      hash: "4742ea1",
+    },
+    "version-info": {
+      beta: {
+        version: "v1.54.0-beta",
+        released: "2025-03-24",
+      },
+      latest: {
+        version: "v1.53.8",
+        released: "2025-03-25",
+        patch: true,
+      },
+      nightly: {
+        version: "v1.52.3",
+        released: "2024-12-16",
+      },
+    },
+  } as const;
+
+  const settings = createMockSettings(updatesSettings);
+  setupPropertiesEndpoints(settings);
+  setupUpdateSettingEndpoint();
+  setupSettingsEndpoints(
+    Object.entries(settings).map(([key, value]) =>
+      createMockSettingDefinition({ key: key as SettingKey, value }),
+    ),
+  );
+
+  renderWithProviders(
+    <>
+      <UpdatesSettingsPage />
+      <UndoListing />
+    </>,
+  );
+};
+
+describe("UpdatesSettingsPage", () => {
+  it("should render a UpdatesSettingsPage", async () => {
+    await act(() =>
```

Comment:
The tests were erroring without the `act`, but I'm not sure why it isn't working here but in https://github.com/metabase/metabase/pull/55064 it wasn't necessary